api-proxy/handlers/|v1|google-account.post.mjs

```diff
@@ -107,7 +107,9 @@ const body = JSON.stringify(
     education: 'NO_EDUCATION',
     disadvantaged: 'PREFER_NOT_TO_ANSWER',
     surveyComments: null
-  }
+  },
+  generalDiscoverySources: [],
```

Comment:
These two properties are required for the Profile object but were not present on our mocks of the get and post endpoints.